salesagility/SuiteCRM
# *WIP Fixed #7298 - Emails 'Bulk Action' is disabled after upgrade to 7.10.16

## Description



Fixes an issue with the bulk action button being disabled when selecting items in the list-view. This fix also fixes pagination and the "selected" counter functionality in the list-view.

## Motivation and Context

Issue reference: #7298 

## How To Test This

1. Check that you can use bulk action after selecting items in the list-view.
2. Check that pagination works with selected items on previous pages.

## Types of changes


### Final checklist
